echopop/tests/conftest.py

```diff
@@ -1,7 +1,13 @@
 import pytest
 from pathlib import Path
 from echopop import Survey
+from _pytest.assertion.util import assertrepr_compare
 
+def pytest_assertrepr_compare( config , op , left , right ):
```

Comment:
Is this a utility function that you call while debugging, or is this intended to be run every time? As of now it does not gets run every time since this function is not called.